### ISSUE TITLE
Password is sent twice from UI when user is created/edited

### ISSUE THREAD

**Issue by fbalak** [Tendrl/ui#873]
In user administration in UI is password sent via API twice when is created new user or edited existing user. API calls that are sent by API are described in https://github.com/Tendrl/api/issues/106.

Steps to Reproduce:
```
1. Log in to Tendrl and open user administration.
2. Open some tool like firebug that shows what API calls are called on site.
3. Create new user.
4. Edit existing user.
```

Actual results:
In API calls sent from UI are 2 parameters: `password` and `password_confirmation`.

Expected results:
There should be only parameter `password` and confirmation that there are provided two identical passwords should be done by frontend. Parameter `password_confirmation` is unnecessary.

**Comment by gnehapk** [comment]
duplicate of #859

**Comment by gnehapk** [comment]
@fbalak Fixed in tendrl-ui-1.6.1-20180317T042235.84aac52.noarch.rpm build. Please verify and close the issue.

**Comment by fbalak** [comment]
Fixed.

Tested with:
```
tendrl-ansible-1.6.1-1.el7.centos.noarch
tendrl-api-1.6.2-1.el7.centos.noarch
tendrl-api-httpd-1.6.2-1.el7.centos.noarch
tendrl-commons-1.6.2-1.el7.centos.noarch
tendrl-grafana-plugins-1.6.2-1.el7.centos.noarch
tendrl-grafana-selinux-1.5.4-2.el7.centos.noarch
tendrl-monitoring-integration-1.6.2-1.el7.centos.noarch
tendrl-node-agent-1.6.2-1.el7.centos.noarch
tendrl-notifier-1.6.0-1.el7.centos.noarch
tendrl-selinux-1.5.4-2.el7.centos.noarch
tendrl-ui-1.6.2-1.el7.centos.noarch
```
